geost/base.py

```diff
@@ -2041,6 +2100,22 @@ def to_qgis3d(
         """
         self.data.to_qgis3d(outfile, relative_to_vertical_reference, **kwargs)
 
+    def to_kingdom(self, outfile, vw=1500.0, vs=1600.0):
```

Comment:
typing staat ook in de functiedefinitie:

`def to_kingdom(self, outfile: str | WindowsPath, vw: float = 1500.0, vs: float = 1600.0`